### PR TITLE
Update PPP - Jonah

### PR DESCRIPTION
**Pull request by jonahtanjz**

### CODE REVIEW

File: docs/team/jonahtanjz.md [hunk 1]
```diff
@@ -1,46 +1,69 @@
...
+    * Highlights: This enhancement requires an additional `FilteredList` and `Predicate`. A new command needs to be added for the user to open the popup on call. During the initialization of the application, it will need to check if there exists expired items. If no expired items are in the inventory, then the popup will not show up. 
+
+  * Ability to **search for food items using description, expiry date, priority and/or tags**.
+    * What it does: This feature allows the user to search for food items using one or more of the above parameters. All food items matching all the specified parameters will be shown.
+    * Justification: This allows the user to be able to quickly and efficiently find food items with common attributes or specific food items, depending on the specificity of their search. 
+    * Highlights: The existing find feature only allows for searching of descripton of food items. This enhancement affects existing commands and commands to be added in future. It required the parser to identify prefixes in the find command and filter the list according to the parameters of the search.
```
Comment (rolandyuwy) [inline docs/team/jonahtanjz.md:33]: I think it should be "requires"

File: docs/team/jonahtanjz.md [hunk 2]
```diff
@@ -1,46 +1,69 @@
...
 
-* **Documentation**:
+  * Added **Priority attribute to food items**.
+    * What it does: Allows the user to specify a priority, either high, medium or low, for a food item.
+    * Justification: This feature provides a convenient way for the user to mark the importance of different food items.
+    * Highlights: This enhancement affects existing commands and commands to be added in future. It required the parser to identify a new prefix and perform necessary validation to ensure that the priority specified in the command is valid.
+
+  * Added a **popup window to show a list of expired food items**.
+    * What it does: If there are expired items in the inventory, a popup will automatically be launched during the start-up of the application, showing a list of expired food items. This popup can also be launch using the 'expired' command.
+    * Justification: This feature alerts the user of expired items in their inventory whenever the application is launched so that they will be reminded to dispose these items and not accidentally consume them. The user can also open this popup using the 'expired' command so that they can access this list at any point of time.
```
Comment (rolandyuwy) [inline docs/team/jonahtanjz.md]: Perhaps use `expired` to standardise with line 54

**Review by rolandyuwy — COMMENTED** [review]
Small suggestions, otherwise LGTM

File: docs/team/jonahtanjz.md (hunk 1 above)
Comment (jonahtanjz) [reply docs/team/jonahtanjz.md:33]: Alright thanks will change.

File: docs/team/jonahtanjz.md (hunk 2 above)
Comment (jonahtanjz) [reply docs/team/jonahtanjz.md]: Ok will change.

**Pull request merged**